src/apps/vpn/ui/screens/initialize/ViewInitialize.qml

```diff
@@ -15,14 +15,25 @@ Item {
     property int safeAreaHeight: window.safeContentHeight
     property string logoSubtitle: MZI18n.ProductDescription
 
-    MZRadialGradient {
+    Rectangle {
+        // This is a fallback for MZRadialGradient
+        color: MZTheme.theme.onBoardingGradient.end
         height: Screen.height
         width: Screen.width
 
         anchors {
             top: parent.top
             topMargin:  -window.safeAreaHeightByDevice()
         }
+    }
+
+    MZRadialGradient {  
+        height: Screen.height
+        width: Screen.width
+        anchors {
+            top: parent.top
+            topMargin:  -window.safeAreaHeightByDevice()
+        }
```

Comment:
If we slap an id on the new fallback rectangle this can just be `anchors.fill: idOfTheFallbackRectangle`.